graphql_compiler/schema_generation/graphql_schema.py

```diff
@@ -320,18 +320,21 @@ def get_graphql_schema_from_schema_graph(schema_graph, class_to_field_type_overr
     for non_graph_cls_name in schema_graph.non_graph_class_names:
         if non_graph_cls_name in hidden_classes:
             continue
+        if not schema_graph.get_element_by_class_name(non_graph_cls_name).abstract:
```

Comment:
Please add an explanation for why we're continuing here. The `hidden_classes` part is obvious, this is less so.

Reply:
There is a comment explaining that the class has to be  abstract at the top of the block.